### PR TITLE
Custom Property editor tutorial - New sample

### PR DESCRIPTION
**Pull request by eshanrnh**
Trello task: https://trello.com/c/PYoO2MQG/47-custom-property-editor-tutorial-new-sample

### CODE REVIEW

File: Tutorials/Creating-a-Property-Editor/index-v9.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+This tutorial covers how to use AngularJS with Umbraco, so it does not cover AngularJS itself. To read about AngularJS, you can take a look at some of the resources here:
+
+- [Egghead.io](https://egghead.io/courses/angularjs-fundamentals)
+- [Angularjs.org/tutorial](https://docs.angularjs.org/tutorial)
+- [Pluralsight](https://www.pluralsight.com/paths/angular-js)
+
+## The End Result
+
+By the end of this tutorial, we will have a suggestion data type running inside of Umbraco, registered as a Data Type in the backoffice and assigned to a Document Type. The data type can create and suggest values.
```
Comment (jonat123) [inline Tutorials/Creating-a-Property-Editor/index-v9.md:28]:
```suggestion
## The End Result

By the end of this tutorial, we will have a suggestion data type running inside of Umbraco, registered as a Data Type in the backoffice, and assigned to a Document Type. The data type can create and suggest values.
```

File: Tutorials/Creating-a-Property-Editor/part-2-v9.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+Let's first add the default value functionality. When the `$scope.model.value` is empty or *undefined*, we want to use the default value. To do that, we add the following to the very beginning of the controller:
+
+```javascript
+if($scope.model.value === null || $scope.model.value === ""){
+    $scope.model.value = $scope.model.config.defaultValue;
+}
+```
+
+and the nat the end we add a getState method:
```
Comment (jonat123) [inline Tutorials/Creating-a-Property-Editor/part-2-v9.md:124]: Is the "nat" on purpose? 

Not sure I completely understand the sentence 😅

**Comment by jonat123** [comment]
@eshanrnh Looks good, i have made a couple of suggestions and comments :-) 

Please take a look when you have time 😊

File: Tutorials/Creating-a-Property-Editor/part-2-v9.md (hunk 2 above)
Comment (eshanrnh) [reply Tutorials/Creating-a-Property-Editor/part-2-v9.md:124]: Oops, Good catch. That line is supposed to be "and then at the end.." just a space typo..

**Pull request merged**